input/images-source/ITI-106-seq.plantuml

```diff
@@ -0,0 +1,17 @@
+@startuml
+hide footbox
+autonumber
+participant "Document Consumer" as DocumentConsumer
+participant "Document Recipient" as DocumentRecipient
+autonumber
+activate DocumentConsumer
+activate DocumentRecipient
+DocumentConsumer -> DocumentRecipient : Generate Metadata (document)
```

Comment:
This should be DocumentSource and not Document Consumer according to text: "The Generate Metadata Option recognizes that there are Document Source Actors that have a structured and coded format of a document"

Reply:
will fix